<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WolvesClass</class>
 <widget class="QMainWindow" name="WolvesClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Wolves</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-20</y>
      <width>600</width>
      <height>400</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>:/Wolves/bg1.png</pixmap>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>更换背景</string>
    </property>
    <addaction name="actionBackground1"/>
    <addaction name="actionBackground2"/>
    <addaction name="actionBackground3"/>
    <addaction name="actionBackground4"/>
    <addaction name="actionBackground5"/>
    <addaction name="actionBackground6"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="actionBackground1">
   <property name="text">
    <string>background1</string>
   </property>
  </action>
  <action name="actionBackground2">
   <property name="text">
    <string>background2</string>
   </property>
  </action>
  <action name="actionBackground3">
   <property name="text">
    <string>background3</string>
   </property>
  </action>
  <action name="actionBackground4">
   <property name="text">
    <string>background4</string>
   </property>
  </action>
  <action name="actionBackground5">
   <property name="text">
    <string>background5</string>
   </property>
  </action>
  <action name="actionBackground6">
   <property name="text">
    <string>background6</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Wolves.qrc"/>
 </resources>
 <connections/>
</ui>
